x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/rule_schemas.schema.yaml

```diff
@@ -203,6 +203,7 @@ components:
         - id
         - rule_id
         - immutable
+        - rule_source
```

Comment:
> I'm therefore not seeing any other valid reason not to make `rule_source` required here other than not to overinflate this PR.

I agree. Since we have the rule params normalization implemented in the rules management client, all rules, after being read from the Alerting framework, are guaranteed to have the `rule_source` field.